src/go/rdctl/cmd/snapshot.go

```diff
@@ -89,12 +90,34 @@ func wrapSnapshotOperation(wrappedFunction cobraFunc) cobraFunc {
 	}
 }
 
-func ensureBackendStarted() error {
+func getSnapshotIdAndManager(args []string) (snapshot.Manager, string, error) {
```

Comment:
This function is used in only one place. It probably makes more sense to put this code where it is used, rather than in a separate function.